### ISSUE TITLE
Running multiactor_environment.py in Jupyter Notebook

### ISSUE THREAD

**Issue by MrChuse** [tensorforce/tensorforce#819]
It raises `TensorforceError: Environment.reset: invalid type <class 'tuple'> != int for  state.`
I checked type of the `state` variable and its zeroth member, it's `<class 'numpy.ndarray'>, <class 'numpy.int32'>`

You could probably reproduce it by copying contents of multiactor_environment.py into freshly created ipynb, copying benchmarks/configs/ppo.json file and referencing it in the notebook. At least that's what I did. 

<details> 
  <summary> Traceback (clickable) </summary>

```
---------------------------------------------------------------------------
TensorforceError                          Traceback (most recent call last)
<ipython-input-6-fa7afb94671c> in <module>
     68 
     69 
---> 70 main()

<ipython-input-6-fa7afb94671c> in main()
     65         max_episode_timesteps=10
     66     )
---> 67     runner.run(num_episodes=1000)
     68 
     69 

D:\Programs\conda\lib\site-packages\tensorforce\execution\runner.py in run(self, num_episodes, num_timesteps, num_updates, batch_agent_calls, sync_timesteps, sync_episodes, num_sleep_secs, callback, callback_episode_frequency, callback_timestep_frequency, use_tqdm, mean_horizon, evaluation, save_best_agent, evaluation_callback)
    610                 else:
    611                     # Check whether environment is ready, otherwise continue
--> 612                     observation = self.environments[n].receive_execute()
    613                     if observation is None:
    614                         self.terminals[n] = self.prev_terminals[n]

D:\Programs\conda\lib\site-packages\tensorforce\environments\environment.py in receive_execute(self)
    373             self._expect_receive = None
    374             if self._num_parallel is None:
--> 375                 states = self.reset()
    376             else:
    377                 parallel, states = self.reset(num_parallel=num_parallel)

D:\Programs\conda\lib\site-packages\tensorforce\environments\environment.py in reset(self, num_parallel)
    510             parallel, states = self._environment.reset(num_parallel=self._num_parallel)
    511         if self._reset_output_check:
--> 512             self._check_states_output(states=states, function='reset')
    513             if self._num_parallel is not None:
    514                 TensorSpec(type='int', shape=(), num_values=self._num_parallel).np_assert(

D:\Programs\conda\lib\site-packages\tensorforce\environments\environment.py in _check_states_output(self, states, function)
    455                 elif not name.endswith('_mask'):
    456                     raise TensorforceError(function + ': invalid component {name} for state.')
--> 457         states_spec.np_assert(
    458             x=_states, batched=(self._num_parallel is not None),
    459             message=(function + ': invalid {issue} for {name} state.')

D:\Programs\conda\lib\site-packages\tensorforce\core\utils\tensor_spec.py in np_assert(self, x, message, batched)
    276             pass
    277         else:
--> 278             raise TensorforceError(
    279                 message.format(issue=('type {} != {}'.format(type(x), self.type)))
    280             )

TensorforceError: Environment.reset: invalid type <class 'tuple'> != int for  state.

```

</details>

**Comment by AlexKuhnle** [comment]
I assume you're using the latest pip version here? The multi-actor feature was only added later, so with the latest Github version it should hopefully work fine. But that's a good point, it's probably time to fix another pip version soon.

**Comment by AlexKuhnle** [comment]
Fixed a new version 0.6.5, so I expect it should now work as expected.
